src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindOutputPresenter.java

```diff
@@ -509,6 +525,33 @@ public void onResponseReceived(String handle)
          }
       });
 
+      dialog.getDirectoryChooser().addValueChangeHandler(new ValueChangeHandler<String>() {
+         @Override
+         public void onValueChange(ValueChangeEvent<String> event)
+         {
+            fileServer_.isGitDirectory(dialog.getDirectory(),
+                                   new ServerRequestCallback<Boolean>() {
+               @Override
+               public void onResponseReceived(Boolean isGitDirectory)
+               {
+                  if (isGitDirectory)
```

Comment:
Nit: GWT has automatic unboxing so this could be simplified to `dialog.setGitStatus(isGitDirectory)`

Reply:
Fixed